src/memory/client/MemoryBrowser.tsx

```diff
@@ -77,21 +77,52 @@ interface State {
   bytesPerRow: number;
   bytesPerGroup: number;
   endianness: 'le' | 'be';
+  childNames: { id: string; name: string }[];
 }
 
 export class MemoryBrowser extends React.Component<Props, State> {
   private addressReq = '';
   private lengthReq = '512';
+  private childReq = '0';
+  public static firtTime: boolean = true;
 
   constructor(props: Props) {
     super(props);
     this.state = {
       bytesPerRow: 32,
       bytesPerGroup: 8,
       endianness: 'le',
+      childNames: [],
     };
   }
 
+  async getAvailableChilds() {
+    if (this.state.childNames.length === 0) {
+      try {
+        const result = await messageBroker.send({
+          command: 'ReadMemory',
+          args: {
+            address: this.addressReq,
+            length: parseInt(this.lengthReq),
+            child: this.childReq,
+          },
+        });
+        let childArray: {
+          id: any;
+          name: any;
+        }[] = [];
+        result.result?.child?.map(function (val: any, index: any) {
```

Comment:
Don't use any if at all possible.

Reply:
I have defined the type for all of them.